### DISCUSSION TITLE
WHIP 1: WiHi Improvement Proposals

### DISCUSSION THREAD

**Discussion by chiumichael** [wihi-labs/WHIP#12] [Ideas]
- Authors: [Mark Ballandies](https://twitter.com/BallandiesMC)
- Start Date: 2023-09-28
- Category: Governance
- Tracking Issue: #2
----
## Summary

A WiHi Improvement Proposal (WHIP) is a (technical) design document providing information to the WiHi community, or describing a new feature for WiHi or its processes or environment. The WHIP should provide a concise (technical) specification of the feature and a motivation for the feature.

We intend WHIPs to be the primary mechanisms for proposing new features, for collecting community input on an issue, and for documenting the design decisions that have gone into WiHi. The WHIP champion is responsible for building consensus within the community and documenting dissenting opinions.

WHIP-1 is sketching a preliminary mechanism for WHIPs to empower the core team to reach design decisions of WiHi more efficiently. It is envisioned that this WHIP will be replaced once the process will be opened for larger parts of the community or shortcomings are identified.

Link to WHIP:

https://github.com/wihi-labs/WHIP/blob/main/0001-wihi-improvement-proposals.md

**Comment by urosvk** [comment]
ack
